lib/task-data/tasks/run-emc-diag.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Run EMC Diag',
+    injectableName: 'Task.Os.Run.Emc.Diag',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        kernelFile: 'vmlinuz-4.1.15-diag',
+        initrdFile: 'ramfs.lzma',
+        kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
+        initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
```

Comment:
looks a little duplication here. Maybe remove 'kernelUri' and 'initrdUri' then add a field like 'directory: common' or 'owner: common' or some other more reasonable name, and more, add 'uri: {{api.server}}' ' is more  flexible?